src/motor/motor_util.cpp

```diff
@@ -241,6 +245,16 @@ int main(int argc, char** argv) {
         motors.insert(motors.end(), tmp_motors.begin(), tmp_motors.end());
     }
     if (ips.size()) {
+        // translate name aliases to ips via json file
+        if (access(json_ip_file.c_str(), F_OK) == 0) {
+            auto motor_ips = nlohmann::json::parse(std::ifstream(json_ip_file));
+            for (auto &address : ips) {
+                if (motor_ips.find(address) != motor_ips.end()) {
+                    address = motor_ips[address].get<std::string>();
+                }
+            }
+        }
```

Comment:
1. If `-j` is specified but we cannot open the file for some reason, can we print out an error and exit instead of continuing silently?
2. If the parsing fails due to being malformed, what happens? Is the error message clear for the user?